src/artm/messages.proto

```diff
@@ -163,6 +163,11 @@ message TopicSelectionThetaConfig {
   repeated float alpha_iter = 3;
 }
 
+// Represents a configuration of a iTopic-like modal Theta regularizer
+message iTopicThetaConfig {
```

Comment:
Do you have potential alternatives for the name of this regularizer?
To follow code style it must start with capital letter -- both here in messages.proto, and also the name of C++ class.
But ``ITopicThetaConfig`` is also bad name -- doesn't look like it tells anything about the purpose of this class. In addtion it collides with C# syntax where ``I`` as a prefix means an interface (e.g. purely abstract class).